source/GameInterface/Services/Kingdoms/Messages/UpdatedKingdomRelation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+using TaleWorlds.CampaignSystem;
+
+namespace GameInterface.Services.Kingdoms.Messages
+{
+    [ProtoContract(SkipConstructor = true)]
+    public record UpdatedKingdomRelation : IEvent
```

Comment:
Also please add summary comments

```suggestion
    public record KingdomRelationUpdated : IEvent
```